contents/dart/Dart.md

```diff
@@ -0,0 +1,228 @@
+### Tooling support
+Dart has out-of-the-box dependency management ([pub.dev](https://pub.dev/)), linting solution ([dartfmt](https://dart.dev/tools/dartfmt)), documentation generator ([dartdoc](https://dart.dev/tools/dartdoc)) and [official testing framework](https://pub.dev/packages/test). After installing Dart SDK, you're ready to go 99% of the time.
+### Easy to learn
+Like many devs who tried Dart says, you might already know Dart. If you already have *any* prior programming experience in any language (say, C, C++, Java, JavaScript, Python, Kotlin, Swift), you're pretty likely to master Dart within weeks for the following reasons:
+* Dart is using a C-style syntax: it won't be a surprise if a function written in Java can be compiled and run in Dart with minor (or no) changes.
+* Dart is a **strongly-typed** language with type-inference support: static typing makes IDE much more helpful and if your program compiles, it most likely works just fine.
+* You may find your favorite syntax (sugar) in Dart, for example
+  * Similar to JavaScript: arrow functions, named parameters (more on that later)
+  * Similar to Kotlin: `get` and `set` keywords for OOP, null safety syntax
+### Versatility
+Dart can run on all major desktop and mobile platforms and can be transpiled to JavaScript for browser execution. It even has three execution modes (again, supporting all major platforms) for the best of all words:
+* for fast development, <tooltip content="Just-in-time">JIT</tooltip> + Dart VM; 
+* for native performance: <tooltip content="Ahead-of-time">AOT</tooltip> compilation to platform-specific instructions; 
+* for browser support: source-to-source compiler to JavaScript
+Moreover, Dart has interoperability with [C](https://dart.dev/guides/libraries/c-interop) and [JavaScript](https://dart.dev/web/js-interop) (i.e. you may execute C or JavaScript code directly from your Dart program). In Flutter, you may also invoke OS-specific methods using [platform channels](https://flutter.dev/docs/development/platform-integration/platform-channels). With regards to this point, Dart is similar to Python as a "glue language".
+### Designed for Declarative UI
+**Dart is a useful declarative UI language.** Dart's syntax allows you to write statically typed code in a JSON-like way, and writing asynchronous code is a breeze in Dart (more on those later).
+
+Of course, like any other language, Dart is [not perfect](https://news.ycombinator.com/item?id=16475074). Even if Dart appears to be open-sourced, it has been pretty internally focused. And unlike Java, C++ or Python, although technically you can write most types of application using Dart, it may not be your safest choice a lot of the times (i.e. you may find yourself writing Dart in a Flutter project most of the time).
```

Comment:
A seperate **limitation** section of sorts would be nice, might be good to add one or two more as well to give a balanced view

Could clarify "internally focused" as well